recipe/meta.yaml

```
@@ -0,0 +1,89 @@
+{% set name = "scrubadub_spacy" %}
+{% set version = "2.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/LeapBeyond/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: cd90477bc3f243d366e5eb8a3a443e58a97e3c1785dcfc4cf02742bc1a4a967e
+  patches:
+    - patches/0001-update-run.py.patch
+    # Created an issue on these 2 failed tests.
+    # https://github.com/LeapBeyond/scrubadub_spacy/issues/6
+    - patches/0002-disabled-test_doc.patch
+    - patches/0003-disabled-test_long_text.patch
+    # tests failed on win by reason: python --version was unexpected at this time.
+    # run nosetests ./tests/ without check on py35
+    - patches/0004-change-py3-on-py-for-win-only.patch  # [win]
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  # https://github.com/AnacondaRecipes/spacy-transformers-feedstock/blob/master/recipe/meta.yaml#L21C1-L22C103
+  # spacy-transformers is not avaliable on s390x
+  skip: True  # [s390x]
+  # spacy-transformers through spacy and pydantic <2.7.4 aren't compatible with python 3.12 currently
+  skip: True  # [py>311]
+
+requirements:
+  build:
+    - patch  # [not win]
+    - m2-patch  # [win]
+  host:
+    - python
+    - pip
+    - wheel
+    - setuptools
+  run:
+    - python
+    - scrubadub >=2.0.0rc0
```

No pre-releases (though post-releases are usually OK)

```suggestion
    - scrubadub >=2.0.0
```